ci/jenkins/jobs/macros.yaml

```diff
@@ -231,6 +231,8 @@
           #!/bin/bash
           set -ex
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          # Delete all kind clusters present from more than 135 mins
+          ./ci/kind/kind-setup.sh destroy --all --until "135m ago"
```

Comment:
can we just make it `--until 135m`? BTW, if you only support "minutes" as the unit, then we should omit it altogether (`--until 135`) and the help message for the option should indicate that the duration is in minutes.